contracts/SafeMathMock.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.4.17;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+// ----------------------------------------------------------------------------
+// Common: SafeMath Library Implementation
+//
+// http://www.simpletoken.org/
+//
+// Based on the SafeMath library by the OpenZeppelin team.
+// Copyright (c) 2016 Smart Contract Solutions, Inc.
+// https://github.com/OpenZeppelin/zeppelin-solidity
+// The MIT License.
+// ----------------------------------------------------------------------------
+
+import "./SafeMath.sol";
+
+
+contract SafeMathMock {
```

Comment:
Do we need "is SafeMath" to inherit SafeMath.sol functionality?

Reply:
the idea is to have this contract call on SafeMath.sol (and inheriting it may tell the compiler to optimise it out)